src/v2-cloud/msger.ts

```
@@ -0,0 +1,274 @@
+  bind<S extends MsgWithConn, Q extends MsgWithOptionalConn>(
+    req: Q,
+    opts: RequestOpts
+  ): ReadableStream<MsgWithError<S>> {
+    const stream = this.raw.bind({ ...req, conn: req.conn || this.conn }, opts);
+    const ts = new TransformStream<MsgWithError<S>, MsgWithError<S>>({
+      transform: (chunk, controller) => {
+        if (!MsgIsTid(chunk, req.tid)) {
+          return;
+        }
+        if (MsgIsConnected(chunk, this.conn)) {
+          if ((opts.waitFor && opts.waitFor(chunk)) || MsgIsError(chunk)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)